packages/ai-ide/src/common/coder-replace-prompt-template.ts

```diff
@@ -1,19 +1,12 @@
+/* eslint-disable @typescript-eslint/tslint/config */
```

Comment:
Question: Will there be enough files in the framework with an MIT license to warrant a change to our eslint check? We could make the MIT license a valid license for the check.

Reply:
Thought about it, but at the moment, I believe we are stable.